acestep/core/generation/handler/service_generate_flow_edit_test.py

```
@@ -0,0 +1,115 @@
+"""Unit tests for the flow-edit dispatch helper (#1156 PR-B).
+
+Regression coverage for codex round-1 P1 + 2×P2 findings:
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)